diff --git a/node_modules/eslint-plugin-import-x/index.d.cts b/node_modules/eslint-plugin-import-x/index.d.cts
index 47b1370..25f1ae9 100644
--- a/node_modules/eslint-plugin-import-x/index.d.cts
+++ b/node_modules/eslint-plugin-import-x/index.d.cts
@@ -1,3 +1,3 @@
-import eslintPluginImportX from './lib/index.js'
+declare const eslintPluginImportX: typeof import('./lib/index.js')
 
 export = eslintPluginImportX
